axiom/tokens_integration_test.go

```diff
@@ -0,0 +1,91 @@
+//go:build integration
```

Comment:
We don't use build tags anymore.

Then again, I just noticed they have been reintroduced partly with monitors and notifiers. Will clean that up, as well.